### PR TITLE
feat: add evaluate langchain chains notebook

### PR DESCRIPTION
**Pull request by eliasecchig**
# Description

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

- [x] Follow the [`CONTRIBUTING` Guide](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/CONTRIBUTING.md).
- [x] You are listed as the author in your notebook or README file.
  - [x] Your account is listed in [`CODEOWNERS`](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/.github/CODEOWNERS) for the file(s).
- [x] Make your Pull Request title in the <https://www.conventionalcommits.org/> specification.
- [x] Ensure the tests and linter pass (Run `nox -s format` from the repository root to format).
- [x] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by eliasecchig** [comment]
Spelling fixed

File: gemini/evaluation/evaluate_langchain_chains.ipynb [hunk 1]
```diff
@@ -0,0 +1,884 @@
+{
```
Comment (inardini) [inline gemini/evaluation/evaluate_langchain_chains.ipynb:1]: <p>Typo in the Overview</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (inardini) [inline gemini/evaluation/evaluate_langchain_chains.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.        for i, row in df.iterrows():
</pre><p>Add docstring to the 1st helper function</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (inardini) [inline gemini/evaluation/evaluate_langchain_chains.ipynb:1]: <p>Can you add a statement here to introduce Model Eval framework? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (inardini) [inline gemini/evaluation/evaluate_langchain_chains.ipynb:1]: <p>Can you add the cleaning section? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

**Review by inardini — APPROVED** [review]

**Pull request merged**